app/models/katello/concerns/content_view_filter_rule_common.rb

```diff
@@ -4,6 +4,7 @@ module ContentViewFilterRuleCommon
       extend ActiveSupport::Concern
 
       included do
+        audited :associated_with => :filter
```

Comment:
@ares, @tbrisker,

I was checking why test cases are failing on this PR. 
This line is causing a problem. Its weird when I remove `:associated_with => :filter` still test cases are failing and on removal of `:audited`, every failing tests run properly without any failure.
Also, I have tried by moving `audited` from `concerns` file to actual model but no change tests are still failing.

I will debug more on that but would like to share findings so far.

Reply:
I think it's collision of audited `#version` method that's being added from lib/audited/auditor.rb while ContentViewFilter has it's own `#version`. I don't see an easy fix for this, so I'd recommend dropping audit definition for this model for now and open a separate issue for it.

Reply:
@ares, 

Yeah, right! It's due to #version.
I was checking that only how I can fix this.
But it's better not to block all audit changes for it.
I will drop audit definition for this model and update the PR. 

Thank you @ares.